### PR TITLE
[#117424027]  create a page that displays app details 

### PR DESCRIPTION
**Pull request by ademola-raimi**
#### What does this PR do?

Displays a page that shows the app details created when the save changes button is clicked on
#### Description of Task to be completed?

The register Application button when clicked on, the next displayed page will be a page displaying the app details just created by the user. This is to certify that the information the user inputted is the correct info and if not, the user can edit the information by clicking on the edit button. This functionality will be dealt with in another PR.
#### How should this be manually tested?

Navigate to http://suyabay.app/developer/myapp/new which is a simple form. Fill the necessary field and hit 'SAVE CHANGES". This will create the app and displays the details of the app. The details displayed are  App's name, Owner's username, app's url and the app's token.
#### Any background context you want to provide?

None
#### What are the relevant pivotal tracker stories?
- #117424027
#### Screenshots (if appropriate)

screen shot 2016-04-20 at 4 11 46 pm
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)

### CODE REVIEW

File: app/Http/Controllers/ApiPagesController.php [hunk 1]
```diff
@@ -41,4 +41,67 @@ public function createNewApp()
...
+
+        AppDetail::create([
+        'name'         => $request->name,
+        'user_id'      => auth()->user()->id,
+        'homepage_url' => $request->homepage_url,
+        'description'  => $request->description,
+        'api_token'    => $this->generateToken(),
+        ]);
+
+        return redirect('/developer/myapp/app-detail');
```
Comment (andela-oadebayo) [inline app/Http/Controllers/ApiPagesController.php]: **Code Quality Nitpick**
@andela-araimi It is always better and looks neater when you pass a route's name to a redirect method. Check this out https://laravel.com/docs/master/routing#named-routes on how to use named routes in your routes file.

File: app/Http/Controllers/ApiPagesController.php [hunk 2]
```diff
@@ -41,4 +41,67 @@ public function createNewApp()
...
+    }
+
+    /**
+     * This method shows the app details created by the user if on session
+     * and display an error message if not on session.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function showAppDetails()
```
Comment (andela-oadebayo) [inline app/Http/Controllers/ApiPagesController.php]: **Code Quality - Critical**
@andela-araimi This method is meant to show the details of one particular app, so this name you are using `showAppDetails` is misleading, something like `showNewAppDetail` will be better.

File: app/Http/Controllers/ApiPagesController.php [hunk 3]
```diff
@@ -41,4 +41,67 @@ public function createNewApp()
...
+    {
+        if (Auth::check()) {
+            $appDetail = AppDetail::where('user_id', auth()->user()->id)->orderBy('created_at', 'desc')->first();
+           
+            return view('api.pages.mynewlyaddedappdetail', compact('appDetail'));
```
Comment (andela-oadebayo) [inline app/Http/Controllers/ApiPagesController.php]: **Code Quality - Critical**
@andela-araimi The view name here `mynewlyaddedappdetail` could be better, (you need to work on naming variables, methods, views and classes) just `newappdetails` should suffice here.

**Comment by unicodeveloper** [comment]
Please rebase. I am still not comfortable with this style and please let the Homepage url values be clickable/linkable

**Comment by unicodeveloper** [comment]
Meet @andela-abankole @andela-fokosun to guide you on a nice design for the list of my apps.

**Comment by unicodeveloper** [comment]
Please also include a link at the right sidebar to let developers go back to the developer api homepage

**Comment by unicodeveloper** [comment]
Please rebase.

**Comment by unicodeveloper** [comment]
A lot still need to be done on this PR @andela-araimi 
1. Change "Save Changes" to "Create App" on the button that appears on the "Create new App" form.
2. The sidebar should have a link that leads to the developer homepage
3. On the create new app form, there should a title/header named "Create new App"
4. When a new app has been created, it should be accompanied with a success message that shows the app has been successfully created.

Please note these points above and fix them ASAP!

File: app/Http/Controllers/ApiPagesController.php [hunk 4]
```diff
@@ -39,7 +39,65 @@ public function myApp()
...
+     * and display an error message if not on session.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function showNewAppDetails()
+    {
+            $appDetail = AppDetail::where('user_id', auth()->user()->id)->orderBy('created_at', 'desc')->first();
+           
+            return view('api.pages.newappdetails', compact('appDetail'));
+        }
```
Comment (unicodeveloper) [inline app/Http/Controllers/ApiPagesController.php]: Please fix all Nitpick CI issues.

**Comment by unicodeveloper** [comment]
What's the latest on this @andela-araimi ?

**Comment by unicodeveloper** [comment]
How does the UI look like now?

**Comment by ademola-raimi** [comment]
Still working on the UI for this. The UI for  `My apps` is ready. Before our code review today, the UI for this will also be ready

**Comment by unicodeveloper** [comment]
In your sentence, the "My apps" should be highlighted so I don't think it's a grammatical error

**Comment by ademola-raimi** [comment]
Noted and corrected.

**Comment by unicodeveloper** [comment]
Please rebase with the latest on staging @andela-araimi

**Pull request merged**